webserver/views/data.py

```diff
@@ -93,7 +93,7 @@ def summary(mbid):
     except db.exceptions.NoDataFoundException:
         summary_data = {}
 
-    if "highlevel" in summary_data:
+    if "highlevel" in summary_data and summary_data["highlevel"]:
```

Comment:
Okay, I get your idea now. Took me way too long to understand it since I'm not used to `.get()`. Updated in 5e017461a5a6a4244d77185721e0dcf46bca2fbc.